### ISSUE TITLE
Documents fonts, chunk type 0x03

### ISSUE THREAD

**Issue by dertseha** [inkyblackness/ss-specs#29]
Document (Import) the format on fonts.
Although this information was not documented in the original ss-specs file, TSSHP had a working fonts implementation.

**Comment by dertseha** [comment]
Fonts are actually quite simple - all header bytes with values are in use and documented.
I'd like to have a working export as well, for instance to the popular [BMFont](http://www.angelcode.com/products/bmfont/) format. If that works, this issue can be closed.

**Comment by dertseha** [comment]
I tried to export to the BMFont format. Documentation is minimal and I found only one tool to show text in a font (Angelcodes own [test viewer](http://www.angelcode.com/dev/bmfonts/)).
I exported to XML as this was described in a blog post - turned out, that viewer doesn't use XML, but a plain text format. After doing that, it still wouldn't show the characters (only white blocks).

The MS Windows bitmap font format could also be an option, though this one is a bit more complex.

I'll leave it. It would have been great, but modifying the fonts is only a very, very nice to have.
